webapp/components/Comment.vue

```diff
@@ -17,12 +17,20 @@
         :callbacks="{ confirm: deleteCommentCallback, cancel: null }"
         style="float-right"
         :is-owner="isAuthor(author.id)"
+        v-on:showEditCommentMenu="editCommentMenu"
       />
     </no-ssr>
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
-    <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
+    <ds-space margin-bottom="small"/>
+    <div v-if="openEditCommentMenu">
+      <hc-edit-comment-form
+        v-bind:comment="comment"
+        v-bind:post="post"
+        v-on:showEditCommentMenu="editCommentMenu"
```

Comment:
Not sure if you want to change, or not, and maybe you already know this, but in case you don't, you can write these
```
:comment="comment"
:post="post"
@showEditCommentMenu="editCommentMenu"
```